pkg/functions/function.go

```diff
@@ -72,6 +72,11 @@ type Function struct {
 
 	//DeploySpec define the deployment properties for a function
 	Deploy DeploySpec `yaml:"deploy"`
+
+	// Enable extensible integrations
+	// These values are ad-hoc and opaque from the view of the functions core
+	// used to comunicate keys through to concrete implementations.
+	Enable []string `yaml:"enable,omitempty"`
```

Comment:
What about we name this `Extension` ?

Reply:
@zroubalik @lkingland  Maybe feature extensions?